### ISSUE TITLE
Replace remaining unittest with PyTest 

### ISSUE THREAD

**Issue by dpgraham4401** [USEPA/haztrak#300]
# 🐞 Bug Report

### TLDR
The title says it all.

We started using PyTest because of the fantastic reusable fixtures and the lower maintenance cost. However, we still have a large number of [unitTest](https://docs.python.org/3/library/unittest.html) based regression tests that use old json fixtures, hacky setup functions (and syntax I know a lot of pythonistas dislike).

The old unittest are usually of lower quality (partially because they're older, partially b/c there's a lot of boilerplate that causes false positives when refactored); let's replace them. You can find the pytest fixtures and a base class for testing API views in [conftest.py](https://github.com/USEPA/haztrak/blob/main/server/apps/trak/tests/conftest.py)

Bonus points for consolidating the JSON directories [here](https://github.com/USEPA/haztrak/tree/main/server/apps/trak/tests/serializers/json) and [here](https://github.com/USEPA/haztrak/tree/main/server/apps/trak/tests/views/json) into a single directory and convert the process of reading those files into fixtures as well (for tests that use the JSON).
